src/setup/platformSetup/index.native.js

```diff
@@ -25,9 +25,6 @@ export default function () {
     // Setup Flipper plugins when on dev
     if (__DEV__) {
         require('flipper-plugin-bridgespy-client');
-        const RNAsyncStorageFlipper = require('rn-async-storage-flipper').default;
```

Comment:
Why is this not needed anymore?

Reply:
1. It was breaking due to peer dependencies and the library not being maintained
2. I believe @marcaaron said this was a debugging tool that is not used anymore

Reply:
It might be used by some (but I haven't used it in a long time). There are workarounds to this package e.g. if you need to know the contents of AsyncStorage you can always log out values. Might be interesting to do our own Flipper plugin for Onyx at some point 🤔